add upgrade instructions (#39)
neo4j/docs-ops-manager: modules/ROOT/pages/upgrade/index.adoc

= Upgrading from an earlier version

> :description: This section describes the upgrade process from an earlier version of NOM.

Reviewer: Switch places, description first, then header.

[[server]]
== Upgrading server

[NOTE]
====

> Before starting the upgrade process, it is recommended to backup your existing NOM installation folder.

Reviewer: ```suggestion
Before starting the upgrade process, it is recommended to back up your existing NOM installation folder.
```

====

[[server-linux]]
=== On Linux

> . Backup any files you've modified inside the current NOM server directory.

Reviewer: ```suggestion
. Back up any files you've modified inside the current NOM server directory.
```

This usually includes `etc/neo4j-ops-manager-server.service` and any files inside `certificates/` directories.
. Download latest NOM server package for your platform.
. Stop NOM server by running `systemctl stop neo4j-ops-manager-server`.
. Extract the downloaded package into the existing installation directory.
. Copy the backed up files back into their corresponding locations.
. Start NOM server by running `systemctl start neo4j-ops-manager-server`.
. Verify that the server is operating as expected by checking logs.

[[server-windows]]
=== On Windows

> . Backup any files you've modified inside the current NOM server directory.

Reviewer: ```suggestion
. Back up any files you've modified inside the current NOM server directory.
```
